src/gui/optionsdialog.cpp

```diff
@@ -1451,6 +1481,36 @@ void OptionsDialog::saveWebUITabOptions() const
     pref->setDynDNSUsername(m_ui->DNSUsernameTxt->text());
     pref->setDynDNSPassword(m_ui->DNSPasswordTxt->text());
 }
+
+void OptionsDialog::on_btnWebUIAPIKeyCopy_clicked()
```

Comment:
You should not be relying on automatic signal connections: call `connect();` manually.
```suggestion
void OptionsDialog::onBtnWebUIAPIKeyCopy()
```

Reply:
This class seems to heavily rely on automatic signal connections. Did we move away from that pattern?

Reply:
> Did we move away from that pattern?

Exactly.

Reply:
Moved to manual `connect()`.